certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json

```diff
@@ -0,0 +1,819 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tesseron ASM Ticket",
+    "description": "This connector allows you to use Tesseron ASM Ticket API to create, edit and search Tickets.",
```

Comment:
@developmentLL the word API is [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info) in the description, please remove it.